string trafficLightId
uint8 evState
uint32 endTime
